extras/label_configs/carrots

```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```

Comment:
Maybe we need a better way to `fix` labels on Github in the event of a change?
Not sure how this would interface considering we don't track what the repo label set is, might be able to do:

```
ash github:labels [fix/change] [labelConfig] [org/repo]
  > Are you sure you want to change the labels in [org/repo] (Y/n): 
```